### PR TITLE
restore full path var on win; rename to CONDA_PREFIX

### LINKED ISSUE #2764: PREFIX environment variable for activated env
Consider setting `PREFIX` and/or `CONDA_PREFIX` as an environment variable for activated envs.

CC @msarahan

### PR DESCRIPTION
**Pull request by msarahan**
Fixes #2764

Please test. Do not just trust me on this.

### CODE REVIEW

File: bin/activate [hunk 1]
```diff
@@ -74,10 +74,10 @@ if (( $? == 0 )); then
...
     fi
-    # CONDA_ENV_PATH is always the full path to the activated environment.  It is not set
+    # CONDA_PREFIX is always the full path to the activated environment.  It is not set
     #    when no environment is active.
     # Legacy support: CONDA_DEFAULT_ENV is either env name or full path if given as path.
-    #    CONDA_ENV_PATH is always the full path, for consistency.
+    #    CONDA_PREFIX is always the full path, for consistency.
     #    Last date of change: 2016-06-21
     firstpath=${PATH%%:*}
     export CONDA_ENV_PATH="$(echo ${firstpath} | sed "s|/bin$||")" &>/dev/null
```
Comment (mwiebe) [inline bin/activate]: This should be `CONDA_PREFIX` to reflect the changed comment, looks like.

Comment (msarahan) [inline bin/activate]: Ugh.  That was an unsaved buffer.  Time to stop for the day.

**Comment by msarahan** [comment]
Thanks for the double-check @mwiebe

File: bin/activate.bat [hunk 2]
```diff
@@ -48,6 +48,9 @@
     SET "PROMPT=(%CONDA_NEW_ENV%) %PROMPT%"
 )
 
+@REM always store the full path to the environment, since CONDA_DEFAULT_ENV varies
+@FOR /F "tokens=1 delims=;" %%i in ("%PATH%") DO @SET "CONDA_PREFIX=%%i"
```
Comment (mwiebe) [inline bin/activate.bat]: I copied this `activate.bat` to `C:\Anaconda3\Scripts`, it doesn't work right:

```
C:\Dev\SequoiaAll>activate sequoia_all_msvc2012rel
prepending C:\\Anaconda3\\envs\\sequoia_all_msvc2012rel and C:\\Anaconda3\\envs\\sequoia_all_msvc2012rel\\Library\\mingw-w64\\bin and C:\\Anaconda3\\envs\\sequoia_all_msvc2012rel\\Library\\usr\\bin and C:\\Anaconda3\\envs\\sequoia_all_msvc2012rel\\Library\\bin and C:\\Anaconda3\\envs\\sequoia_all_msvc2012rel\\Scripts to PATH

(sequoia_all_msvc2012rel) C:\Dev\SequoiaAll>echo %CONDA_PREFIX%
C:\Windows\system32
```

Comment (mwiebe) [inline bin/activate.bat]: I think replace `PATH` with `NEW_PATH` here and it will work.

Comment (mwiebe) [inline bin/activate.bat]: Yes, using `NEW_PATH` fixes it for me.

**Comment by mwiebe** [comment]
Sweet, are you going to push out a quick release immediately, or should we proceed with finding a workaround for now before applying a proper fix when the next release is out?

**Pull request merged**